{"levelname": "ERROR", "asctime": "2024-12-03T14:57:26.003Z", "name": "processing", "message": "Unable to publish collection version.", "lineno": 81, "pathname": "/single-cell-data-portal/backend/layers/processing/publish_revisions.py", "collection_id": "2fc350a2-7a08-41ca-a07f-74add2d2875c", "collection_version_id": "1316689f-a537-4607-95f5-21a356611c28", "errors": ["error message"]}
